services/api/src/middlewares/error-handler.js

```diff
@@ -1,18 +1,18 @@
-const Sentry = require('@sentry/node');
-
 module.exports = async (ctx, next) => {
   try {
     await next();
   } catch (err) {
+    let { status = 500, message, details } = err;
 
-    const status = Number.isInteger(err.status) ? err.status : 500;
-    const body = err.isJoi ? err.details[0].message : err.message;
-
-    if (status === 500) {
-      Sentry.captureException(err);
+    if (details) {
+      message = details.map((d) => d.message).join('\n');
```

Comment:
this looks fishy to me details might not be defined, details is something an joi error will have only ?

Reply:
yes that's why the `if` block. It only applies to Joi.

Reply:
details might be an object => you don't know what the error object is gonna look like. mongoose or elasticsearch might throw an error that has details, or whatever other library we might include in the future.

The details is specify to joi errors.

Reply:
okok fair enough... I was thinking we could standardize on `details` based on Joi for other error types but probably too soon for that... I just put back the isJoi